### ISSUE TITLE
Search within files

### ISSUE THREAD

**Issue by woodsb02** [thedillonb/CodeHub#112]
When a source file has been opened for viewing, it is not currently possible to search for text within that file.

This would be handy, as finding what you are looking for in a long source file can be difficult without searching.

**Comment by oomek** [comment]
Same here. I was very extatic when I found this app, but after finding out that the source listing does not have a simple search function my extasy went from wow to, it's ok but limited.

**Comment by aunkrig** [comment]
Yeah; reading and editing code without a „search“ feature is not very useful. Certainly CODEHUB is not an IDE, but today it's not even possible to make even small changes.
